packages/stack/src/types.tsx

```diff
@@ -313,6 +313,10 @@ export type StackNavigationOptions = StackHeaderOptions &
       bottom?: number;
       left?: number;
     };
+    /**
+     * Function that given `style` returns a React Element to display as a overlay.
+     */
+    overlay?: (style: StyleProp<ViewStyle>) => React.ReactNode;
```

Comment:
can we change it to an object with `style` property?
```suggestion
    overlay?: (style: StyleProp<ViewStyle>) => React.ReactNode;
```

Reply:
yes, it will be more intuitive